### PR TITLE
.coafile: Add YAMLLintBear

### LINKED ISSUE #448: Check repository for .yml files using YAMLLintBear
Add YAMLLintBear in .coafile and fix .yml file issues raised by coala. See https://github.com/coala/coala-bears/pull/1799 for reference.
difficulty/newcomer

Opened via [gitter](https://gitter.im/coala/coala/?at=5945ca5d6462d8493c224171) by @Monal5031

### PR DESCRIPTION
**Pull request by Arsfiqball**
Fixes #448 

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: .coafile [hunk 1]
```diff
@@ -21,3 +21,7 @@ bears = InvalidLinkBear
 [rstcheck]
 bears = RSTcheckBear
 files = **.rst
+
+[yml]
+bears = YAMLLintBear
+files = *.yml
```
Comment (adhikasp) [inline .coafile]: I think it's more safe to just write `**.yml` here for future proofing.

Comment (Arsfiqball) [inline .coafile]: Oh ya, I missed it.

**Comment by Arsfiqball** [comment]
Hi @adhikasp , thank you for reviewing. You taught me important thing. :+1:

**Comment by jayvdb** [comment]
ack 8c15537

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**